scripts/report/report.py

```diff
@@ -109,7 +109,7 @@ def get_header(Self):
         return header
 
     def run_script(self):
```

Comment:
`report.sh` prints all the errors (FileNotFoundError as well) to stderr and it needs to be thrown away